src/Varieties/Func/Preservation.lagda

```diff
@@ -196,7 +177,7 @@ module _ {α : Level}{𝒦 : Pred (SetoidAlgebra α α) (ov α)} where
  P⨅𝒜 = piso (pprod PAi) (⨅≅ λ _ → ≅-sym Lift-≅)
   where
   PAi : (i : I) → Lift-Alg (𝒜 i) (ov α)(ov α) ∈ P{α}{ov α} 𝒦
-  PAi i = pbase ∥ i ∥
+  PAi i = {!!} -- pbase ∥ i ∥
```

Comment:
Are you sure you want to check in something with a hole?

Reply:
good point.. this is taken care of by the latest commit (entitled "updates")